WorkingWithText/Program.cs

```diff
@@ -40,15 +141,56 @@ public static bool IsValidTime(string hyphenNum)
         // Trim off unneeded spaces.
         public static string PascalConverter(string aFewWords)
         {
-        	return default;
+        	if (string.IsNullOrWhiteSpace(aFewWords))
+            {
+                return null;
+            }
+
+            var seperatedWords = aFewWords.Trim().Split(' ');
+
+            //capitlize first letter of each word
+
+            var result = string.Empty;
+
+            foreach (var eachWord in seperatedWords)
+            {
+                //make every letter lowercase
+                var trimmedWord = eachWord.Trim().ToLower();
+
+                //makes the first letter uppercase and assigns a new variable due to string memory immutability
+                var trimmedCap = trimmedWord[0].ToString().ToUpper() + trimmedWord.Substring(1);
+
+                //list of capped words
+                result += trimmedCap;
+            }
+            return result;
         }
 
         // 5- Write a method that accepts an English word. Count the number of vowels
         // (a, e, i, o, u) in the word. So, if the user enters "inadequate", the program should
         // return 6.
         public static int VowelCounter(string aWord)
         {
-            return default;
+            var wordArr = aWord.ToLower().ToCharArray();
+
+            char[] vowelArray = { 'a', 'i', 'u', 'e', 'o' };
+            int result = 0;
+            //checks for vowels
+            foreach (var letter in wordArr)
+            {
+                if (letter == 'a')
+                    result++;
+                else if (letter == 'e')
+                    result++;
+                else if (letter == 'i')
+                    result++;
+                else if (letter == 'o')
+                    result++;
+                else if (letter == 'u')
+                    result++;
```

Comment:
This works, there's ways you could make this shorter or more readable. A switch statement, an if statement with a bunch of `or`'s, or `"aeiou".contains(letter)` are also options. Doesn't matter for something like checking for vowels in a words but it could matter down the road.